Include/internal/pycore_frame.h

```diff
@@ -210,6 +210,13 @@ PyGenObject *_PyFrame_GetGenerator(_PyInterpreterFrame *frame)
     return (PyGenObject *)(((char *)frame) - offset_in_gen);
 }
 
+
+static inline bool
+_PyFrame_IsIncomplete(_PyInterpreterFrame *frame)
```

Comment:
Can you add a comment here explaining what "incomplete" means in this context (I know the code "explains" it, but I think it can benefit from some more elaborate explanation).